CP-CPS.md

```diff
@@ -314,7 +314,7 @@ No stipulation.
 
 ### 4.3.1 CA actions during certificate issuance
 
-At a high level, the following steps are taken during issuance of a Subscriber Certificate. ISRG's automated processes confirm that all names which will appear in the Common Name and/or list of SANs of the certificate have been properly validated to be controlled by the Subscriber requesting the certificate. The to-be-signed certificate is linted, then signed by a Subordinate CA in an HSM. After issuance is complete, the certificate is stored in a database and made available to the Subscriber.
+At a high level, the following steps are taken during issuance of a Subscriber Certificate. ISRG's automated processes confirm that all requested names have been properly validated to be controlled by the Subscriber requesting the certificate. The to-be-signed certificate is linted, then signed by a Subordinate CA in an HSM. After issuance is complete, the certificate is stored in a database and made available to the Subscriber.
```

Comment:
Is there any required content for this section? I think we could just delete this paragraph. BRs have:

> 4.3 Certificate issuance
> 4.3.1 CA actions during certificate issuance
> 4.3.1.1 Manual authorization of certificate issuance for Root CAs
> Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA
system operator, system officer, or PKI administrator) to deliberately issue a direct command in
order for the Root CA to perform a certificate signing operation.
> 4.3.1.2 Linting of to‑be‑signed Certificate content
Due to the complexity involved in implementing Certificate Profiles that conform to these
Requirements, it is considered best practice for the CA to implement a Linting process to test the
technical conformity of each to‐be‐signed artifact prior to signing it.

It seems like we should delete the paragraph we have here, _add_ a 4.3.1.1 documenting issuance from Root CAs, and _add_ section discussing our linting process.

Also, I noticed this language in our CPS:

> The to-be-signed certificate is linted

This is potentially confused with tbsCertificate. We should mirror the BR language around what we lint (and also document that we lint using a "dummy" key, and lint both precertificates and final certificates).

Reply:
I'd prefer not to line up with section 4.3.1.1 and 4.3.1.2 unless we have to: those sub-sub-subsections don't appear in the RFC 3647 outline, and could be renumbered by the BRs at any time.

We just added the linting language here in https://github.com/letsencrypt/cp-cps/pull/232. I would argue that we _do_ lint the tbsCertificate, by virtue of signing it with a dummy key. I don't think we necessarily need to add more detail than that at this time, since all the rest of the detail in the BRs section 4.3.1.2 is suggestions.

Finally, we shouldn't discuss linting final certificates here, as that is covered in Section 8.7.

Reply:
I think you're totally right that this section can be stripped down much further. But that feels like a task separate from our current incident remediation. I've filed https://github.com/letsencrypt/cp-cps/issues/247 to track this, is that okay?

Reply:
Works for me 👍🏻